docs/source/commands/supergraphs.mdx

```diff
@@ -57,41 +57,9 @@ You can also pass config via stdin:
 cat ./supergraph.yaml | rover supergraph compose --config -
 ```
 
-### From a Studio variant
```

Comment:
Moved this section down since I feel explaining the supergraph.yaml takes precedence